components/card/card.css

```diff
@@ -0,0 +1,119 @@
+.card-container {
+  background-color: var(--ZINC-COLOR-LIGHT);
+  display: flex;
+  max-width: 25rem;
+  border: thin solid var(--BORDER-COLOR);
+  border-radius: 1.5rem;
+  padding: 1rem;
+  position: relative;
+}
+
+.card-dismiss {
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.text-card {
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.card-vertical {
+  flex-direction: column;
+  max-width: 20rem;
+  padding: 1rem;
+  min-height: 20rem;
+}
+
+.card-horizontal {
+  min-width: 30rem;
+  padding: 1rem;
+  max-height: 15rem;
+}
+
+.card-heading {
+  padding: 8px;
```

Comment:
use rem instead of px.

Reply:
hey, I am using rem for anything above 10 px.
Thanks for the review @Ananyamadhu08